source/client-side-encryption/tests/fle2-CreateCollection.yml

```diff
@@ -602,6 +602,18 @@ tests:
         object: database
         arguments:
           collection: "encryptedCollection"
+          encryptedFields: {
+                "escCollection": "encryptedCollection.esc",
+                "eccCollection": "encryptedCollection.ecc",
+                "ecocCollection": "encryptedCollection.ecoc",
+                "fields": [
+                    {
+                        "path": "firstName",
+                        "bsonType": "string",
+                        "keyId": { "$binary": { "subType": "04", "base64": "AAAAAAAAAAAAAAAAAAAAAA==" }}
+                    }
+                ]
+            }
```

Comment:
To test the expectation, this should be in the `command_started_event` below.

I suggest adding a YAML anchor to reference the encryptedFields used in create. Here is one proposal: https://github.com/mongodb/specifications/compare/master...kevinAlbs:add-encryptedFields-to-expectations?expand=1#diff-36980134f3bcb9a64658afc5058a1d6b49fa472a09486fccee2d8ab1a71fad6b

Reply:
@kevinAlbs Ah, that’s much better even! Thank you, done :sparkles: